DOCUMENTATION.md

```diff
@@ -2190,12 +2190,42 @@ console.log(value);
 
 ## 8.7. SQL
 
-The SQL service provided by Hazelcast Node.js client allows you to query data stored in `IMap` declaratively.
+The SQL service provided by Hazelcast Node.js client allows you to run SQL queries.
 
-> **WARNING: The SQL feature is currently in beta. The compatibility between versions is not guaranteed. API might change between versions without notice. While in beta, SQL feature is tested against the same version of the IMDG, e.g 4.2.x
-> client is tested against 4.2.x IMDG server.**
+> **WARNING: The SQL feature have become stable in 5.0. In order a client and a server to be fully compatible with each other, their major
+> versions must be the same.**
 
-### Example: How to Query an IMap using SQL
+> **WARNING: A [Unisocket Client](#822-unisocket-client) must connect to a member that is not
+> [lite](https://docs.hazelcast.com/hazelcast/latest/management/cluster-utilities.html#enabling-lite-members) (non-lite members are called data members),
+> otherwise Data Manipulation Language(DML) statements below won't work for that client.**
+
+### 8.7.1 SQL Statements
+
+#### Data Manipulation Language(DML) Statements
+
+- [SELECT:](https://docs.hazelcast.com/hazelcast/latest/sql/select.html) Read data from a table.
+- [SINK INTO/INSERT INTO:](https://docs.hazelcast.com/hazelcast/latest/sql/sink-into.html) Ingest data into a map and/or forward data to other systems.
+- [UPDATE:](https://docs.hazelcast.com/hazelcast/latest/sql/update.html) Overwrite values in map entries.
+- [DELETE:](https://docs.hazelcast.com/hazelcast/latest/sql/delete.html) Delete map entries.
+
+####  Data Definition Language(DDL) Statements
+
+- [CREATE MAPPING:](https://docs.hazelcast.com/hazelcast/latest/sql/create-mapping.html) Map a local or remote data object to a table that Hazelcast can access.
+- [SHOW MAPPINGS:](https://docs.hazelcast.com/hazelcast/latest/sql/show-mappings.html) Get the names of existing mappings.
+- [DROP MAPPING](https://docs.hazelcast.com/hazelcast/latest/sql/drop-mapping.html) Remove a mapping.
+
+#### Job Management Statements
+
+- [CREATE JOB:](https://docs.hazelcast.com/hazelcast/latest/sql/create-job.html) Create a job that is not tied to the client session.
+- [ALTER JOB:](https://docs.hazelcast.com/hazelcast/latest/sql/alter-job.html) Restart, suspend, or resume a job.
+- [SHOW JOBS:](https://docs.hazelcast.com/hazelcast/latest/sql/show-jobs.html) Get the names of all running jobs.
+- [DROP JOB:](https://docs.hazelcast.com/hazelcast/latest/sql/drop-job.html) Cancel a job.
+- [CREATE OR REPLACE SNAPSHOT (Enterprise only):](https://docs.hazelcast.com/hazelcast/latest/sql/create-snapshot.html) Create a snapshot of a running job, so you can stop and restart it at a later date.
+- [DROP SNAPSHOT (Enterprise only):](https://docs.hazelcast.com/hazelcast/latest/sql/drop-snapshot.html) Cancel a running job.
```

Comment:
Does it cancel the job or the snapshot?

Reply:
Again, copy paste, so I think we should change the Hazelcast docs if it's wrong